packages/amplify-graphql-maps-to-transformer/src/__tests__/graphql-maps-to-transformer.test.ts

```diff
@@ -274,4 +279,13 @@ describe('@mapsTo directive', () => {
     expect(attachInputMappingSlot_mock).not.toBeCalled();
     expect(attachResponseMappingSlot_mock).not.toBeCalled();
   });
+
+  it('pre-mutates the schema to reassign type mappings', () => {
```

Comment:
adding one test increased the coverage from 58 -> 80? Thats nice!

Reply:
Yup, small package w/ few branches means each function and codebranch can have an outsized impact. :D